requirements/test.in

```diff
@@ -11,3 +11,4 @@ pycodestyle
 pytest-cov                # pytest extension for code coverage statistics
 pytest-django             # pytest extension for better Django support
 responses
+twine
```

Comment:
Could you add a comment explaining what we use it for?

Reply:
will do.